sphinx/source/gui.rst

```diff
@@ -363,15 +363,15 @@ documented above, the following variables exist:
 
 .. var:: gui.hover_color = '#3284d6'
 
-    The color used by focused items in the gui, including the text of
+    The color used by focused items in the GUI, including the text of
     of buttons and the thumbs (movable areas) of sliders and scrollbars.
 
 .. var:: gui.selected_color = '#555555'
 
     The color used by the text of selected buttons. (This takes priority
     over the hover and idle colors.)
 
-.. var:: gui.insensitive_color = '#8888887f'
```

Comment:
Ah, could be. But I just followed the description in the same document:

`These variables should generally be set to hexadecimal color codes, which are strings of the form "#rrggbb"`